src/styles/_variables.scss

```diff
@@ -81,7 +81,7 @@ $container-lg: (975px + $gutter-width);
 $container-xl: (1381px + $gutter-width);
 
 // MATERIAL DESIGN
-$md-icon-size: 24px;
+$md-icon-size: 18px;
```

Comment:
not following
https://projects.invisionapp.com/d/#/console/14289248/297268561/preview